frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -351,52 +350,38 @@ function Dashboard(props: DashboardProps) {
     cancelFetchDashboardCardData();
   });
 
-  const renderContent = () => {
-    if (!dashboard) {
-      return null;
-    }
```

Comment:
We already have this check [here](https://github.com/metabase/metabase/pull/52639/files#diff-2dd6656adc7b65513709719ab362c7364ea8614ce00940d13241f2918e946d06R397-R399).